src/Dhall/Tutorial.hs

```diff
@@ -168,6 +168,16 @@ module Dhall.Tutorial (
 import Data.Vector (Vector)
 import Dhall
 
+-- $setup
+--
+-- >>> :set -XOverloadedStrings
+-- >>> writeFile "makeBools" "λ(n : Bool) → [ n && True, n && False, n || True, n || False ]"
```

Comment:
Wait, where does the file go? I can’t find anything about the context in which doctests are run in its documentation.

Reply:
doctests are run from the root folder of the project, so the file is added to the root foldr

Reply:
Do you think we could switch to a `TMP` folder first?

Reply:
I tried that and it doesn't work.  `ghci` unloads all modules when you `:cd` to another directory so it breaks `doctest`

Reply:
As a side note, I think technically you can run `doctest` in a temporary directory, but you would need to then specify `-i` accordingly so that `doctest` will find your sources.